### ISSUE TITLE
Workers Default To Off On Restart

### ISSUE THREAD

**Issue by NickDavison** [Codaone/DEXBot#365]
## Expected Behavior

Workers retain same on/off status they had when program terminated.
## Actual Behavior

Workers must be manually reconfigured to on status each time.

## Steps to Reproduce the Problem

  1. Turn workers on
  2. Close program
  3.

## Specifications

  - Version: 0.7.9
  - OS: Windows 10 Pro

**Comment by NickDavison** [comment]
Would be great to get this changed, as this will mean the DexBot could be scheduled to run via task scheduler, or run from startup preventing downtime from system reboots, windows updates, etc.

**Comment by thehapax** [comment]
@MarkoPaasila close issue as we have #366 referenced?

**Comment by bitfag** [comment]
@NickDavison for automated running I suggest to use cli version of dexbot. It will just run all of the configured workers.

I considering restoring workers "running" state after GUI startup a bad idea because it cannot be expected behavior in 100% cases. As a clean solution, we may consider adding a per-worker setting like "Auto-start this worker on GUI launch".

**Comment by MarkoPaasila** [comment]
I think such a case where the user has some workers running and some not, a temporary situation, and the likely goal for all users is to have all workers running. Or is there a point in having a worker not running?

**Comment by bitfag** [comment]
@MarkoPaasila Yes there is a point. Market conditions may change so user would like to temporarily disable worker (think relative orders). Cli user can edit config and comment out worker, but GUI user?

**Comment by thehapax** [comment]
@MarkoPaasila i agree. there exists cases where you don't need to run workers, especially when testing or switching between various strategies. There should be an config option in the GUI that allow for "run all" or "run only selected" 

moved this status to available if someone wants to implement.
